### ISSUE TITLE
Option --hpx:queuing=local-priority-lifo is not configured

### ISSUE THREAD

**Issue by diehlpk** [STEllAR-GROUP/hpx#6519]
## Expected Behavior

Command line option `--hpx:queuing=local-priority-lifo` worked with HPX 1.9.1 on Arm. 

## Actual Behavior

`Command line option --hpx:queuing=local-priority-lifo is not configured in this build. Please make sure 128bit atomics are available.`

## Steps to Reproduce the Problem

... Please be as specific as possible while describing how to reproduce your problem.

  1. Compile Octo-Tiger with HPX master from today

## Specifications

  - HPX Version: Master
  - Platform (compiler, OS): gcc 12

**Comment by hkaiser** [comment]
@diehlpk Yes, this is expected. We have disabled the schedulers that depend on 128bit atomics on platforms where those are either not available or are not lockfree. We have seen large performance regressions in those cases (especially on ARM). 

We're currently working in a fix, at least for x64 systems. ARM would be another matter to be addressed separately.
